src/components/property-items/condominium-items.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { IconDefinition } from '@fortawesome/fontawesome-svg-core';
+import { ItemsSection } from './items-section';
+import {
+  faConciergeBell,
+  faCouch,
+  faElevator,
+  faPlay,
+  faSwimmingPool,
+  faTshirt,
+  faVolleyballBall,
+} from '@fortawesome/free-solid-svg-icons';
+import { fetchProperties } from '../../api/fetch-properties';
+import { LoadingError } from '../loading/loading-error';
+import { CondominiumItemTitle } from './items-enums';
+
+const condominiumItemsIcons: { [key in CondominiumItemTitle]: IconDefinition } = {
```

Comment:
Geralmente esses objetos com `key` vc pode tipar usando `Record<key, value>`. Dessa forma, provavelmente vc nao vai precisar fazer o cast na linha 33.

```suggestion
const condominiumItemsIcons: Record<CondominiumItemTitle, IconDefinition> = {
```